Add Getting Started Section (#84)
AlexsLemonade/refinebio-docs: docs/getting_started.md

> ## Getting Started with refine.bio dataset

Reviewer: _a_ refine.bio dataset?

> refine.bio dataset includes gene expression matrices and experiment and sample metadata for the samples that you selected for download.

Reviewer: ```suggestion
A refine.bio dataset includes gene expression matrices and experiment and sample metadata for the samples that you selected for download.
```

### Structure

> The structure of your download folder is slightly vary depending on whether you chose to aggregate by experiment or by species.  

Reviewer: ```suggestion
The structure of your download folder varies slightly depending on whether you chose to aggregate by experiment or by species.  
```

* Individual gene expression matrices and their corresponding sample metadata files are in their own directories.

* Gene expression matrices are the tab-separated value (TSV) files named by the experiment accession number (if aggregated by experiment) or species name (if aggregated by species).
Note that samples are _columns_ and rows are _genes_ or _features_.
This pattern is consistent with the input for many programs specifically designed for working with high-throughput gene expression data but may be transposed from what other machine learning libraries are expecting.

* Sample metadata (e.g. disease vs. control labels) are contained in TSV files with `metadata` in the filename as well as any JSON files.
We apply light harmonization to some sample metadata fields, which are denoted by `refinebio_` (`refinebio_annotations` is an exception).
The contents of a sample's `refinebio_annotations` field include the submitter-supplied sample metadata.

* Experiment metadata (e.g., experiment title and description) are contained in JSON files with `metadata` in the filename.

> Please see the [Downloadable Files](http://docs.refine.bio/en/latest/main_text.html#downloadable-files) section for more details.

Reviewer: ```suggestion
Please see the [Downloadable Files](http://docs.refine.bio/en/latest/main_text.html#downloadable-files) of our documentation section for more details.
```

Maybe? Since this is split off now. Hard for me to visualize what it'll look like/the flow.

### Usage

> The gene expression matrix TSV and JSON files can be read in, manipulated, or parsed with standard functions or libraries in the language of your choice. Below are some code snippets to help you import the data into R or Python and examine it.

Reviewer: One line per sentence
